tests/conftest.py

```diff
@@ -683,26 +683,123 @@ def product_level_constant_forecast_w_quantiles(hierarchical_structure):
             "timestamp": ["2000-01-05", "2000-01-06"] * 4,
             "segment": ["a"] * 2 + ["b"] * 2 + ["c"] * 2 + ["d"] * 2,
             "target": [1, 1] + [2, 2] + [3, 3] + [4, 4],
-            "target_0.25": [1, 1] + [2, 2] + [3, 3] + [4, 4],
-            "target_0.75": [1, 1] + [2, 2] + [3, 3] + [4, 4],
-        }
+            "target_0.25": [1 / 2, 1 / 4] + [1, 1 / 2] + [2, 1] + [3, 2],
+            "target_0.75": [2, 3] + [3, 4] + [4, 5] + [5, 6],
+        },
+        dtype=float,
     )
     df = TSDataset.to_dataset(df=df)
     ts = TSDataset(df=df, freq="D", hierarchical_structure=hierarchical_structure)
     return ts
 
 
+@pytest.fixture
+def product_level_constant_forecast_w_target_components(hierarchical_structure):
```

Comment:
Both `with` and `without` starts with `w`))

Reply:
Follow the naming of the existing fixtures)

Reply:
Fixed